packages/fxa-auth-server/lib/routes/subscriptions/mozilla.ts

```diff
@@ -44,7 +44,7 @@ export const mozillaSubscriptionRoutes = ({
           strategy: 'oauthToken',
         },
         response: {
-          schema: validators.subscriptionsMozillaSubscriptionsValidator,
+          schema: validators.subscriptionsMozillaSubscriptionsValidator as any,
```

Comment:
There's quite a few of these, the reason they came up is because I pushed up the `joi` version, and hapi sometimes bugs out on some forms of the schema here now. So passing it as `any` ended up being required. This could likely be resolved if we moved up farther and switched to using `joi` instead of the deprecated `@hapi/joi`, but we haven't gotten there yet.

Reply:
Not yet, perhaps after we move to the latest `joi` and see how that changes things yet again.